models/backbones/resnet.py

```diff
@@ -0,0 +1,49 @@
+import torchvision.models as models
+
+from .abstract_backbone import AbstractBackbone
+
+
+def layer_out_filters(resnet_layer):
```

Comment:
please, add docstring like:
def add(a, b):
    """
    Add two numbers and return the result.

    Args:
        a (int): The first number.
        b (int): The second number.

    Returns:
        int: The sum of the two numbers.
    """
    return a + b

or any other docstring format